ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__multi-nightly.yaml

```diff
@@ -990,6 +990,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi
+- as: azure-ipi-sharednetwork-ovn-ipv4-subnet-amd-f28-destructive
+  cron: 25 23 27 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-azure-ipi-sharednetwork-ovn-ipv4-subnet
+- as: azure-ipi-sharednetwork-ovn-ipv4-subnet-arm-f14
```

Comment:
@jinyunma can you help add a z-stream upgrade job for the install in openshift-openshift-tests-private-release-4.18__multi-nightly-4.18-upgrade-from-stable-4.18.yaml, thanks.

Reply:
Done